src/main/java/school/lemon/changerequest/java/introduction/hw1/Task5.java

```diff
@@ -3,10 +3,10 @@
 
 public class Task5 {
     public static double calculateA(double x, double y, double z) {
-        return 0;
+        return ((2 * Math.sin(x - Math.PI / 6) * calculateB(z)) / (1 / 2.0 + (Math.pow(Math.sin(y), 2))));
```

Comment:
It's quite hard to read such expressions. It'll be easier for understanding if you'll split it into some pretty parts, like `top`-part of expression, `bottom`-part of expression.

But, anyway, it do pass all the tests.